ui/raidboss/data/04-sb/trial/lakshmi-ex.txt

```diff
@@ -11,96 +11,99 @@ hideall "Alluring Arm"
 
 ### Adds
 0.0 "Start" sync /:Engage!/ window 0,1
-1.0 "--sync--" sync /:2146:Dreaming Kshatriya/ window 1,10
-15.5 "Inner Demons" # end of castbar
-15.5 "Stotram" # end of castbar
+4.0 "Vril" sync /:Dreaming Kshatriya:2146:/ window 10,10
+12.1 "Tail Slap" sync /:Dreaming Kshatriya:258C:/
+16.0 "Stotram" sync /:Lakshmi:2147:/
+16.9 "Inner Demons" sync /:Dreaming Kshatriya:258D:/
+28.1 "Inner Demons" sync /:Dreaming Kshatriya:258D:/
+29.0 "Tail Slap" sync /:Dreaming Kshatriya:258C:/
+
 # infinite loop
 60.0 "--sync--" sync /.*/ window 0,30 jump 30
 
 ### Warmup
 # first auto comes out before rp text.
-100.0 "--sync--" sync /:Lakshmi:2157:/ window 100,0
-111.0 "Hand Of Grace" duration 11.5 # hat -> disappearing
-114.0 "The Pull Of Light" # end of castbar
-125.7 "Stotram" # end of castbar
-136.7 "The Pull Of Light" # end of castbar
-139.5 "The Path Of Light" duration 5.5 # hat -> cleave
-154.5 "Alluring Arm" # end of castbar
-157.0 "Hands Of Grace/Beauty" duration 11.5 # hat -> disappearing
-161.5 "The Pull Of Light" # end of castbar
-171.7 "Stotram" # end of castbar
-179.0 "The Pall Of Light" # end of castbar
-188.2 "The Pull Of Light" # end of castbar
+107.2 "Hand Of Grace" sync /:Lakshmi:214C:/ window 110,0
+114.4 "The Pull Of Light" sync /:Lakshmi:215E:/
+121.4 "Blissful Arrow" sync /:Lakshmi:214F:/
+122.0 "Blissful Spear" sync /:Lakshmi:2151:/
+124.6 "Stotram" sync /:Lakshmi:2147:/
+135.8 "The Pull Of Light" sync /:Lakshmi:215E:/
```

Comment:
Any chance of adding clarifications on Pull/Pall/Path as translations, to match Normal mode?

Reply:
Done!